src/Resource/utils.ts

```diff
@@ -336,3 +340,50 @@ export async function deprecateSelfResource(
     throw error;
   }
 }
+
+export async function getIncomingLinks(
+  orgLabel: string,
+  projectLabel: string,
+  selfUrl: string,
+  paginationSettings: PaginationSettings,
+): Promise<PaginatedList<ResourceLink>> {
+  try {
+    const { from, size } = paginationSettings;
+    const view = await getSparqlView(orgLabel, projectLabel);
```

Comment:
Can you use the static method `SarqlView.get` in order to avoid 1 api call?

Reply:
They're the same